core/selection.js

```diff
@@ -11,6 +11,10 @@
 		fillingCharSequenceRegExp = new RegExp( fillingCharSequence + '( )?', 'g' ),
 		isSelectingTable;
 
+	function _isWidget( element ) {
```

Comment:
Actually there isn't a need to prefix such function with `_` as it's not a method. `isWidget` would be enough.